src/app.ts

```diff
@@ -0,0 +1,25 @@
+import * as express from "express";
```

Comment:
This is a big change and I'd suggest it's addressed in a completely separate PR and branch to avoid conflicts with other PR's.